cloudslang-cli/src/main/java/io/cloudslang/lang/cli/SlangCLI.java

```diff
@@ -147,6 +147,12 @@ private void registerEventHandlers() {
         handlerTypes.add(ScoreLangConstants.EVENT_INPUT_END);
         handlerTypes.add(ScoreLangConstants.EVENT_OUTPUT_START);
         handlerTypes.add(ScoreLangConstants.EVENT_OUTPUT_END);
+        handlerTypes.add(ScoreLangConstants.EVENT_BRANCH_START);
```

Comment:
async loop is not marked in CLI output.. you just see the task names (so yes, like a regular loop)

you would see something like this:

```
- print_values
- print_values
- print_values
```